cmd/frontend/internal/httpapi/internal.go

```diff
@@ -576,6 +578,35 @@ func serveGitExec(w http.ResponseWriter, r *http.Request) error {
 	return nil
 }
 
+// gitServiceHandler are handlers which proxy git clone requests to the
+// gitserver for the repo.
+type gitServiceHandler struct {
+	GitServer interface {
```

Comment:
Nit (but secretly "please this is a hard requirement"):

```suggestion
	Gitserver interface {
```

Since we also don't use `git_server`.